README.md

```diff
@@ -108,85 +111,64 @@ All commands listed below are of the Linux variety, and assume you already
-
-`./dcrctl --wallet getnewaddress dex`
-
-Alternatively, you can get a deposit address during registration right after
-creating your wallet, from the wallets view link at the top right of the screen.
+External wallet software is not required for most assets, just the DEX client!
+The native light wallets are simplest and best option for most users. But if
+using external wallets, they should be running and synced before staring DEX.
 
-#### Creating a trading wallet for Bitcoin
+Unless you use Decrediton to start DEX, you will need a web browser to open the
+DEX client user interface as described in the next section.
 
-For Bitcoin, you can create a trading wallet using the bitcoin-cli utility.
-**bitcoin-cli** will be included in the quick-start installation as well. You
-can replace `dex` with whatever name you want. The rest of the instructions
-will assume you chose `dex`.
+### Optional External Software
 
-`./bitcoin-cli createwallet dex`
+Depending on which assets you wish to use, you have different choices for wallet
+software. There are native/built-in light wallet options for Bitcoin and Decred,
+an external light wallet option for Litecoin, and full-node support for all
+other assets including: Bitcoin, Decred, Litecoin, ZCash, Dogecoin, Bitcoin
+Cash. The following release will include Ethereum support with a native light
+wallet.
 
-It is recommended that you password-protect your Bitcoin trading wallet.
-We'll use `read` to prevent echoing the password.
+1. For Bitcoin, [Bitcoin Core v0.21 or v22 or v23](https://bitcoincore.org/en/download/) (bitcoind or bitcoin-qt) wallet. Descriptor wallets are not supported in v0.21. Alternatively, you man use the native light wallet.
+2. For Decred, [dcrd](https://github.com/decred/dcrd) and [dcrwallet](https://github.com/decred/dcrwallet), installed from the [v1.7.x release binaries](https://github.com/decred/decred-release/releases), or built from the `release-v1.7` branches. Alternatively you may use Decrediton or the dcrwallet application in SPV mode, or the native light wallet.
+3. For Litecoin, [Litecoin Core v0.21.x](https://litecoin.org/). Alternatively,
+you may use an external Electrum-LTC wallet with the name "default_wallet".
+4. For Dogecoin, [Dogecoin Core v1.14.5+](https://dogecoin.com/).
+5. For ZCash, [zcashd v5.1](https://z.cash/download/).
+6. For Bitcoin Cash, [Bitcoin Cash Node v24+](https://bitcoincashnode.org/en/)
 
-`read -s BTCPASS`
-
-Type your password and hit enter, then do
-
-`./bitcoin-cli -rpcwallet=dex encryptwallet $BTCPASS`
-
-You'll also want to instruct **bitcoind** to load the wallet at startup. Modify
-your **bitcoin.conf** file, located in the **~/.bitcoin** directory on Linux,
-**%APPDATA%\Bitcoin** on Windows, and **~/Library/Application Support/Bitcoin**
-on Mac OS. Open the file in a text editor and add the following line at the end
-of the file.
-
-`wallet=dex`
+See the [wiki](../../wiki/Testnet-Testing) for details on preparing the wallets.
```

Comment:
I didn't change this sentence, but I don't think we should link from the README to this Testnet Testing wiki page for wallet setup.  Will remove this line I think.

Also, the entire "Testnet Testing" wiki page is quite outdated.  There are multiple wallet options, so if we want to maintain some of the content from that page, we should rename it to Full Node Setup Example.